src/js/events/MarathonActions.js

```diff
@@ -47,6 +50,37 @@ module.exports = {
       };
     },
     {delayAfterCount: Config.delayAfterErrorCount}
+  ),
+
+  fetchDeployments: RequestUtil.debounceOnError(
```

Comment:
Let's also get some tests for this new action. Here is a good example to look at: https://github.com/dcos/dcos-ui/blob/master/src/js/events/UsersActions.js